source/blog/2016-08-26-up-and-running-with-ovirt-4-0-and-gluster-storage.html.md

```diff
@@ -44,7 +44,7 @@ I'm running gdeploy from my first host:
 ```
 [host1]# yum install -y epel-release
 
-[host1]# yum install -y https://copr-be.cloud.fedoraproject.org/results/jasonbrooks/gdeploy/epel-7-x86_64/00442697-gdeploy/gdeploy-2.0-0.noarch.rpm ansible1.9
```

Comment:
By removing ansible1.9 from the line, the newer ansible 2.2 is installed by default. I don't recall for sure but I *think* it needed 1.9 specifically.